templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_edit_master.html.twig

```diff
@@ -258,7 +258,13 @@
                 </div>
             {% endif %}
 
-            {% set isAnotherBlueprintSetAsCustomerMaster = (templateVars.isCustomerMasterBlueprintExisting == true and (proceduresettings.master != true or proceduresettings.customer == null))  %}
+            {#
+                A procedure is considered "not a customer master" if either:
```

Comment:
1. A procedure or a procedure template?
2. I think that `proceduresettings.master == true` only means that it is a procedure template (so "it is not marked as master" would never be false for a procedure template), and while I was wondering about this and started investigating, I found there's a `masterTemplate` (boolean) property that we could maybe just check here? (Because `proceduresettings` is actually the `procedure` 🙈 )